.github/workflows/ci.yml

```diff
@@ -0,0 +1,307 @@
+      - name: (🐧 Linux) Install packages fuse2, postgre-${{ env.postgre-version }}
+        if: >-
+          steps.python-changes.outputs.run == 'true'
+          && startsWith(matrix.os, 'ubuntu-')
+        run: >
+          sudo apt-get -y install
+          libfuse2
+          fuse
+          desktop-file-utils
+          postgresql-${{ env.postgre-version }}
+          libxkbcommon-x11-0 libxcb-icccm4 libxcb-image0
+          libxcb-keysyms1 libxcb-randr0 libxcb-render-util0
+          libxcb-xinerama0 libxcb-xfixes0 x11-utils
+
+      - name: (🍎 macOS) Install macfuse
+        if: >-
+          steps.python-changes.outputs.run == 'true'
+          && startsWith(matrix.os, 'macos-')
+        run: brew install --cask macfuse
+
+      - name: (🏁 Windows) Install WinFSP
+        if: >-
+          steps.python-changes.outputs.run == 'true'
+          && startsWith(matrix.os, 'windows-')
+        shell: bash
+        run: |
+          choco install -y --limit-output winfsp --version=${{ env.winfsp-version }}
+          mkdir winfsp-test
+          curl -L https://github.com/billziss-gh/winfsp/releases/download/v1.8/winfsp-tests-1.8.20304.zip -o winfsp-test/winfsp-tests.zip
+          unzip winfsp-test/winfsp-tests.zip -d winfsp-test
+          pwd
+          echo "$(pwd)\winfsp-test" >> "$GITHUB_PATH"
+
+      - uses: ./.github/actions/setup-rust
+        if: steps.python-changes.outputs.run == 'true'
+        with:
+          version: ${{ env.rust-version }}
+          cache-key: python
+        timeout-minutes: 10
+
+      - uses: ./.github/actions/setup-python-poetry
+        if: steps.python-changes.outputs.run == 'true'
+        id: setup-python
+        with:
+          python-version: ${{ env.python-version }}
+          poetry-version: ${{ env.poetry-version }}
+
+      - name: Install python deps
+        if: steps.python-changes.outputs.run == 'true'
+        run: poetry install -E core -E backend
+
+      - name: Debug python & pip version
+        if: steps.python-changes.outputs.run == 'true'
+        shell: bash
+        continue-on-error: true
+        run: |
+          set -x
+          poetry run python --version
+          poetry run pip --version
+
+      - name: Basic tests
+        if: steps.python-changes.outputs.run == 'true'
+        run: poetry run pytest ${{ env.pytest-base-args }} tests -n auto
+        timeout-minutes: 10
+
+      - name: (🐧&🏁 Not macOS) Mountpoint tests
+        if: >-
+          steps.python-changes.outputs.run == 'true'
+          && startsWith(matrix.os, 'macos-') != true
+        run: poetry run pytest ${{ env.pytest-base-args }} tests --runmountpoint --runslow -m mountpoint
+        timeout-minutes: 10
+
+      - name: (🐧 Linux) Install pytest-xvfb plugin for pytest for Qt Testing
+        if: >-
+          steps.python-changes.outputs.run == 'true'
+          && startsWith(matrix.os, 'ubuntu-')
+        run: poetry run pip install pytest-xvfb
+
+      - name: (🐧&🏁 Not macOS) GUI tests
```

Comment:
```suggestion
      - name: (🐧 & 🏁 Not macOS) GUI tests
```

Reply:
I've removed the & entirely then (less clutter this way)